Frameworks/GLKit/GLKViewController.mm

```diff
@@ -91,13 +101,12 @@ - (void)viewWillDisappear:(BOOL)animated {
 }
 
 - (BOOL)_renderFrame {
-    [_calendar setToNow];
-    WFDateTime* dt = [_calendar getDateTime];
-    int64_t frameTime = dt.universalTime;
+    _calendar.SetToNow();
+    WF::DateTime frameTime = _calendar.GetDateTime();
 
-    _timeSinceFirstResume = static_cast<NSTimeInterval>(frameTime - _firstStart) * 0.0000001;
-    _timeSinceLastResume = static_cast<NSTimeInterval>(frameTime - _lastStart) * 0.0000001;
-    _timeSinceLastUpdate = static_cast<NSTimeInterval>(frameTime - _lastFrame) * 0.0000001;
+    _timeSinceFirstResume = timeSpanToInterval(frameTime - _firstStart);
+    _timeSinceLastResume = timeSpanToInterval(frameTime - _lastStart);
+    _timeSinceLastUpdate = timeSpanToInterval(frameTime - _lastFrame);
```

Comment:
The timestamps are offsets from the Windows epoch, which is fixed, so I'd be surprised if localization affected them at all.

I haven't done any serious tests on this, though. I just translated the code as-is.